src/components/hack-details.js

```diff
@@ -92,10 +97,13 @@ export function HackDetails({ match, dispatch, userId, hackDetails, history }) {
                 />
               </Grid>
             </Grid>
+            <Grid item xs={10} className={classes.creator}>
+              <Typography variant="body1">Created by: {hackDetails.creator ? hackDetails.creator.name : "Unknown"}</Typography>
```

Comment:
Change "Unknown" to "Legends tell of a legendary hacker, whose origins are unknown."

Reply:
P.S. I'm kidding